api/src/pcapi/routes/backoffice/templates/offer/list.html

```diff
@@ -169,10 +168,9 @@ <h2 class="fw-light">Offres individuelles</h2>
                     {% endif %}
                   </td>
                 {% endif %}
-                {% if has_permission("MANAGE_OFFERS_AND_VENUES_TAGS") %}
-                  <td>{{ row.tags | format_as_badges | safe }}</td>
-                  <td>{{ offer.rankingWeight | empty_string_if_null }}</td>
-                {% endif %}
+                {% if has_permission("MANAGE_OFFERS_AND_VENUES_TAGS") %}<td>{{ row.tags | format_as_badges | safe }}</td>{% endif %}
+                <td>{{ links.build_author_name_to_details_link(offer.author.id, offer.author.full_name) }}</td>
```

Comment:
`authorId` est `nullable` dans `Offer`, donc je crains fort une exception lorsqu'on visualise une offre sans `author`.
```suggestion
                <td>{% if offer.author %}{{ links.build_author_name_to_details_link(offer.author.id, offer.author.full_name) }}{% endif %}</td>
```